cmd/flux/create_tenant.go

```diff
@@ -65,7 +65,6 @@ var (
 )
 
 func init() {
-	createTenantCmd.Hidden = true
 	createTenantCmd.Flags().StringSliceVar(&tenantNamespaces, "with-namespace", nil, "namespace belonging to this tenant")
 	createTenantCmd.Flags().StringVar(&tenantClusterRole, "cluster-role", "cluster-admin", "cluster role of the tenant role binding")
```

Comment:
```suggestion
	createTenantCmd.Flags().StringVar(&tenantClusterRole, "cluster-role", "admin", "cluster role of the tenant role binding")
```

Prevents users from being deleting their own namespace, or fiddle with resource quotas.

Reply:
No that's no good, admin doesn't allow access to custom resources.

Reply:
Also a tenant can't delete the namespace, for that to happen it needs to apply it first and the service account doesn't allow that.